styles.css

```diff
@@ -90,3 +105,184 @@ h2 span {
   font-size: 25px;
   color: #505f79;
 }
+
+#main-head {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  text-align: center;
+  color: #28352f;
+}
+
+#main-head h2 {
+  font-size: 28px;
+}
+
+.work-div {
+  text-align: center;
+  display: flex;
```

Comment:
[Optional] For best CSS practices and to make your code DRY, consider creatting a class of `d-flex` and applying it to the index.html. Currently `display: flex;` is appearing 9 times in your code base. You can read more about DRY CSS [here.](https://teamtreehouse.com/library/dry-css)

Reply:
- [ ] Check line 161 of your CSS file and remove the commented code.

Reply:
Thank you for your review. Everything is solved